project/api/models.py

```diff
@@ -101,6 +102,21 @@ def save(self, *args, **kwargs):  # pylint: disable=W0221
         super(PoliticalRelation, self).save(*args, **kwargs)
 
 
+class CachedData(models.Model):
+    """
+    Stores cache of ranked Wikidata entities
+    """
+
+    wikidata_id = models.PositiveIntegerField(unique=True)  # Excluding the Q
+    location = models.PointField()
+    date = models.DateField()
+    rank = models.PositiveIntegerField()
```

Comment:
Forgot to write this in my previous review: A `type` field is missing. Right now the only types we have are `battle` and `treaty`